database/migrations/2024_04_11_083805_create_pembelian_tikets_table.php

```diff
@@ -13,10 +13,11 @@ public function up(): void
     {
         Schema::create('pembelian_tikets', function (Blueprint $table) {
             $table->id();
+            $table->string('code')->nullable();
             $table->integer('id_users');
             $table->integer('id_destinasi');
             $table->integer('jumlah_tiket');
-            $table->integer('total_harga');
+            $table->integer('harga');
```

Comment:
```suggestion
            $table->integer('Total_harga');
```
nama kolomnya jangan diganti